### PR TITLE
Add big ol' list of recommended libraries

### PR DESCRIPTION
**Pull request by nie7321**
## Overview

Moses asked if we could add a list of recommended libs/tools, so here's a first stab at that.

1. Let me know if you think anything here is UNSUITABLE for inclusion (take-backs are hard)
1. Feel free to suggest (or just directly add) any other libs for inclusion

## TODO
- [x] Merge lang-specific languages into the application patterns page(s)
- [x] jQuery vs vanilla JS vs Vue strategy
- [x] Linting for js

## Checklist

- [x] Proof-reading
- [x] Any diagrams/figures have their source files included

### CODE REVIEW

**Review by cecolby11 — APPROVED** [review]

**Review by bab4379 — APPROVED** [review]
Looks good to me

**Comment by nie7321** [comment]
I want to revise this into a couple different stack pages instead of just a list, so popping this back to draft

**Comment by nie7321** [comment]
I had a note from @cecolby11 too on the frontend framework section -- I should add some guidance on when to do jQuery vs. Vue, similar to some other sections had

**Comment by nie7321** [comment]
@bab4379 @cecolby11 I broke the recommendations out into their own "X stack" pages with more specific & focused information, plus a generic frontend page.

I am the laravel shill so that's the most detailed page. The express one needs some love, and the Spring one is bare. Can you think of anything I should add to those?

**Review by bab4379 — APPROVED** [review]

**Review by cecolby11 — APPROVED** [review]
looks good to me, I think the express stack page is good for now as express is not particularly prescriptive about how you use it or what you do with the rest of your node app, though I will keep thinking about it.

**Comment by cecolby11** [comment]
express middleware perhaps? 

guidance around linting might also be useful (I don't think we already have that in the cloud docs) - in the context of this PR, adding recommended linting packages and configs? It's happened a lot lately that using a linter would have saved a dev a ton of time.

**Comment by nie7321** [comment]
> express middleware perhaps?

@cecolby11 Do you mean stuff like the `load-ssm-secrets` one? I should probably package that one up since it seems broadly useful 😞 

> guidance around linting might also be useful

Good call!

I haven't mentioned linting/static analysis much because I'm hoping we can pick up [StyleCI](https://styleci.io/).

File: development/express-stack.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+| [serverless-http for Express](https://github.com/dougmoscrop/serverless-http)                          | ExpressJS Lambda/API Gateway adapter |
+| [Font Awesome](https://fontawesome.com/)                                                               | Icons                                |
+| [moment.js](https://momentjs.com/)                                                                     | DateTime & timezone manipulation     |
+| [libphonenumber](https://github.com/google/libphonenumber#third-party-ports)                           | Phone number info & validation       |
+| [Axios](https://github.com/axios/axios)                                                                | HTTP client                          |
+| [node-oracledb](https://github.com/oracle/node-oracledb) <sup>*</sup>                                  | Oracle DB driver                     |
+| [@sentry/node](https://www.npmjs.com/package/@sentry/node) †                                           | Error logging                        |
+| AirBnB eslint rules                                                                                    | Code style                           |
+
```
Comment (cecolby11) [inline development/express-stack.md:35]:
```suggestion
| [body parser](https://github.com/expressjs/body-parser)                                                | Express middleware to parse HTTP request body| 
```

**Pull request merged**